contracts/FlowIDTableStaking.cdc

```diff
@@ -740,8 +740,20 @@ pub contract FlowIDTableStaking {
 
                 if nodeRecord.tokensStaked.balance == 0.0 || nodeRecord.role == UInt8(5) { continue }
 
+                var rewardPortion: UFix64 = 0.0
+
                 /// Calculate the amount of tokens that this node operator receives
-                let rewardAmount = FlowIDTableStaking.epochTokenPayout * (nodeRecord.tokensStaked.balance / totalStaked)
+                if totalStaked >= 100000000.0 {
+                    // Maximum UFix64 divisor is 100M so we need to scale the numbers
+                    // in order to not cause overflow
+                    let div1000dividend = nodeRecord.tokensStaked.balance / 1000.0
+                    let div1000divisor = totalStaked / 1000.0
+                    rewardPortion = div1000dividend / div1000divisor
```

Comment:
This is the accepted solution for now, but will probably change in the future. Though I don't know if it would be better to use 1000 or 100 as the scaling factor. I posted a question in #flow-computation yesterday about it but didn't get a response.